python/version_label/version_label.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+# 
+# CONFIDENTIAL AND PROPRIETARY
+# 
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your 
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from sgtk.platform.qt import QtCore, QtGui
+
+from .ui import resources_rc
+
+class VersionLabel(QtGui.QLabel):
```

Comment:
This class is called **VersionLabel**, but it appears to accept any type of shotgun data. Perhaps it should only accept **Version** type data and bail if given something else?